ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs

```diff
@@ -5,8 +5,43 @@
 
 namespace ReactNative.Bridge
 {
-    static class DispatcherHelpers
+    internal static class DispatcherHelpers
     {
+        private static Dispatcher _dispatcher;
+
+        internal static Dispatcher CurrentDispatcher
```

Comment:
The DispatcherHelpers now have a static property that will hold the dispatcher used by the static methods... The dispatcher must be explicitly set...